### PR TITLE
Ember 3.27 Blueprint Updates (Drop Ember 3.16 Support)

### PR DESCRIPTION
**Pull request by elwayman02**
* Runs ember-cli-update for Ember 3.27
* [Breaking] Drop Ember 3.16 support (per blueprint)
* Update GitHub Actions to match new ember-try scenarios
* Bump yarn version in GitHub action config
* Add deprecation workflow and silence deprecations
* Drop IE11 from compile targets
* Configure test files for qunit eslint plugin

### CODE REVIEW

File: tests/dummy/config/targets.js [hunk 1]
```diff
@@ -6,12 +6,20 @@ const browsers = [
...
+//
+// If you need IE11 support on a version of Ember that still offers support
+// for it, uncomment the code block below.
+//
+// const isCI = Boolean(process.env.CI);
+// const isProduction = process.env.EMBER_ENV === 'production';
+//
+// if (isCI || isProduction) {
+//   browsers.push('ie 11');
+// }
```
Comment (ygongdev) [inline tests/dummy/config/targets.js:22]: Is this intended to be commented out?

File: README.md [hunk 2]
```diff
@@ -13,8 +13,8 @@ We adhere to the [Ember Community Guidelines](https://emberjs.com/guidelines/) f
 Compatibility
 ------------------------------------------------------------------------------
 
-* Ember.js v3.8 or above
-* Ember CLI v2.13 or above
+* Ember.js v3.20 or above
+* Ember CLI v3.20 or above
```
Comment (ygongdev) [inline README.md:17]: Just curious, what change caused this compatibility requirement to be upgraded?

File: tests/dummy/config/targets.js (hunk 1 above)
Comment (elwayman02) [reply tests/dummy/config/targets.js:22]: Yup!

File: README.md (hunk 2 above)
Comment (elwayman02) [reply README.md:17]: https://github.com/ember-cli/ember-cli/pull/9491

**Review by ygongdev — APPROVED** [review]

**Pull request merged**